src/components/Card/Card.tsx

```diff
@@ -18,6 +18,44 @@ interface CardProps {
   image?: React.ReactNode;
   /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
   modifiers?: string[];
+  /** Layout **/
+  layout?: "vertical" | "horizontal";
+  /** Border */
+  border?: false | true;
+  /** Padding */
+  padding?: string;
+}
+
+// CardImage child-component
+export function CardImage(props) {
+  const { src, alt } = props;
+  if (src) {
+    return <Image src={src} alt={alt} />;
+  } else {
+    return null;
+  }
+}
+
+// CardHeading child-component
+export function CardHeading(props) {
+  const { children, id, level, url } = props;
```

Comment:
Also include `displaySize` and `className` props.  Note: `displaySize` is a new prop that was recently added to `Heading`.